tests/drivers/spi/spi_loopback/boards/nucleo_f411re.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1 {
+	dmas = <&dma2 5 3 0x28440 0x03
```

Comment:
we need a set of macro's/defines to make the magic numbers easier to read (can be a follow up PR).